spring-cloud-dataflow-docs/pom.xml

```diff
@@ -168,6 +168,8 @@
 									<backend>pdf</backend>
 									<sourceHighlighter>rouge</sourceHighlighter>
 									<attributes>
+										<pdf-stylesdir>${project.basedir}/src/main/asciidoc/pdf-theme</pdf-stylesdir>
```

Comment:
We could move the theme out of the `asciidoc` directory (maybe as a sibling `asciidoc-pdf-theme`) so that it does not end up in the `generated-docs` dir